cpp/src/graph.cpp

```diff
@@ -0,0 +1,67 @@
+#include "../include/graph.h"
+
+#include <fstream>
+#include <sstream>
+#include <iostream>
+
+using namespace std;
```

Comment:
Using 

```C++
using namespace std; 
```

is discouraged. I think if we use range-based for loops, writing std:: all the time will not be as painful.